app/tray/Account/Requests/TransactionRequest/AdjustFee/index.js

```diff
@@ -266,10 +266,11 @@ class TxFeeOverlay extends Component {
       return limitGasUnits(rawGasLimit)
     }
 
-    const receiveValueHandler = (value, name, setAsHex = true) => {
+    const receiveValueHandler = (value, name) => {
+      // FIXME: should not set state directly
```

Comment:
Agree, this is a linting error, would be nice to splat it

Reply:
I'm not sure, I guess we could just change it to use `setState` but the problem is keeping the state in this component isn't quite right to begin with -- it disconnects the values from those on the actual request. one symptom of that could be seen in this reported issue where the gas limit was being changed to 21000 in the component and the actual value that was being set on the request, ~130000, wasn't being reflected

Reply:
I didn't want to go down a long refactor rabbit hole and instead decided to keep this as is until after the release. I can spend a little more time on it though if you think it would be impactful